### PR TITLE
v0.15

### PR DESCRIPTION
**Pull request by lpil**
Waiting on release of v0.15 for merge

### CODE REVIEW

File: cheatsheets/gleam-for-python-users.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ In Gleam all strings are UTF-8 encoded binaries.
 
 ### Tuples
 
-Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `tuple("a", "b")` - can be confused for a function call, which is not!
+Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `#("a", "b")` - can be confused for a function call, which is not!
```
Comment (tranzystorek-io) [inline cheatsheets/gleam-for-python-users.md:427]: The new tuple syntax most probably can't be confused for a function call 😛

File: cheatsheets/gleam-for-rust-users.md [hunk 2]
```diff
@@ -417,11 +417,11 @@ let (_, password, _) = my_tuple;
 
 #### Gleam
 
-Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `tuple("a", "b")` - can be confused for a function call, which is not!
+Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `#("a", "b")` - can be confused for a function call, which is not!
```
Comment (tranzystorek-io) [inline cheatsheets/gleam-for-rust-users.md:420]: Same as above

File: cheatsheets/gleam-for-erlang-users.md [hunk 3]
```diff
@@ -459,7 +459,7 @@ All strings in Gleam are UTF-8 encoded binaries.
 ### Tuples
 
 Tuples are very useful in Gleam as they're the only collection data type that allows for mixed
-types of elements in the collection. The syntax for a tuple literal - `tuple("a", "b")` - can be
+types of elements in the collection. The syntax for a tuple literal - `#("a", "b")` - can be
```
Comment (tranzystorek-io) [inline cheatsheets/gleam-for-erlang-users.md:462]: The new tuple syntax most probably can't be confused for a function call 😛

Comment (lpil) [reply cheatsheets/gleam-for-erlang-users.md:462]: Well spotted, thanks!

File: cheatsheets/gleam-for-elixir-users.md [hunk 4]
```diff
@@ -480,7 +480,7 @@ In both Elixir and Gleam all strings are UTF-8 encoded binaries.
 
 ### Tuples
 
-Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `tuple("a", "b")` - can be confused for a function call, which is not!
+Tuples are very useful in Gleam as they're the only collection data type that allows mixed types in the collection. The syntax for a tuple literal - `#("a", "b")` - can be confused for a function call, which is not!
```
Comment (tranzystorek-io) [inline cheatsheets/gleam-for-elixir-users.md:483]: One more obsolete tuple note 😅

Comment (lpil) [reply cheatsheets/gleam-for-elixir-users.md:483]: Well spotted

**Pull request merged**